### ISSUE TITLE
options are not saved.

### ISSUE THREAD

**Issue by sysoryrag** [dungeons-of-moria/umoria#28]
Sorry if it is an error on my part in compiling.

out of source compile of version 5.7.11

use = to change rogue like commands from yes to no
restart and rogue like commands is set to yes again.

in attempt to find the problem I tried the following with the corresponding results.

umoria -n creates a new game with rogue like set to yes
umoria -r opens game with rogue like set to yes

If it is an error on my part please advise.

I thank you for your work on this great project.

truly,
--gary

**Comment by mrcook** [comment]
Hi sysoryrag, thanks for the report, this does seem to be a bug! Sorry about that. I'll fix it as soon as I can.

**Comment by sysoryrag** [comment]
Thank you very much.  I am so pleased that I can still play Moria and I
appreciate all that you and your team do.

Truly,
Gary Rosys

**Comment by mrcook** [comment]
Hi Gary, I've now made a fix for the keys issue (https://github.com/dungeons-of-moria/umoria/commit/8dc1e4eb377257968d9a5dba84b1a628861e3b6b)

I've also removed the `-r` flag from the CLI options. As the settings are stored in the game save file I think having that CLI flag adds some confusion, so I think it's better to just have them set in one place. The game will however still default to roguelike keys for a new game.

Thanks again for flagging this!

Michael
